tests/components/cambridge_audio/test_media_player.py

```diff
@@ -127,6 +128,29 @@ async def test_entity_supported_features(
         in attrs[ATTR_SUPPORTED_FEATURES]
     )
 
+async def test_entity_supported_features_with_controlBus(
+    hass: HomeAssistant,
+    mock_stream_magic_client: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test entity attributes with control bus state."""
+    await setup_integration(hass, mock_config_entry)
+    
+    mock_stream_magic_client.state.pre_amp_mode = False
+    mock_stream_magic_client.state.control_bus_mode = ControlBusMode.AMPLIFIER
```

Comment:
```suggestion
    mock_stream_magic_client.state.control_bus = ControlBusMode.AMPLIFIER
```
This should fix the test.